src/Runner.Listener/JobDispatcher.cs

```diff
@@ -346,7 +346,38 @@ private async Task RunAsync(Pipelines.AgentJobRequestMessage message, string orc
                 }
 
                 var term = HostContext.GetService<ITerminal>();
-                term.WriteLine($"{DateTime.UtcNow:u}: Running job: {message.JobDisplayName}");
+
+                string workflowName = string.Empty;
+                if(message.Variables.TryGetValue("system.workflowFilePath", out var workflowPath))
```

Comment:
this might not work if the job is coming from a callable workflow since `system.workflowFilePath` is always pointed to the entry workflow.

Reply:
In that case, the name of the entry workflow will be displayed, ie the workflow that calls the reusable workflow. Also, the job name from the entry workflow will be displayed, as well as the job name from the reusable workflow (entry workflow job name / reusable workflow job name). Here is an example of output in case of reusable workflow:
```
2022-03-29 07:36:15Z: Running job: "call-workflow / first_job" (workflow "use_callable_workflow.yml" in repository "ruvceskistefan/git-runner-test")
```
`call-workflow` is job name in entry workflow,
`first_job` is job name in callable workflow,
`use_callable_workflow.yml` is entry workflow and
`ruvceskistefan/git-runner-test` is repository name for entry workflow.

In my opinion this works properly, because the user can easily check which reusable workflow is called from entry workflow.